genie-web/src/main/java/com/netflix/genie/web/spring/autoconfigure/dtos/DtosAutoConfiguration.java

```diff
@@ -0,0 +1,102 @@
+/*
```

Comment:
I'd rename this to something else.. `ServerBinding` maybe? `ServerAddressInfo`?
Because it's not really configuring anything about DTOs, it's just creating 2 objects that happen to be DTOs (but could also not be..)

Reply:
Yeah. I've generally tried to name the auto configurations to match the "module" that they're tied to but you bring up a good point about these two objects. They probably should be moved as they're more "introspection" or "metadata" type classes meant for runtime information about the web server. Let me add another commit that moves them and this config somewhere else.